packages/network/src/models/networks.ts

```diff
@@ -0,0 +1,32 @@
+import { Network } from '@openzeppelin/defender-sdk-base-client';
+
+export type NetworkType = 'production' | 'test';
```

Comment:
Didn't we add `forked` type?

Reply:
Good catch! This is specific to the `listSupportedNetworks` endpoint, which only lists prod/test networks. We have a separate endpoint to list forked networks.